cmd/forwarder/http.go

```diff
@@ -224,6 +224,14 @@ func (s *httpServer) process(req *http.Request, r io.Reader, remoteAddr string,
 		s.pMetadataLogsReceived.Inc(numLogs)
 	}
 
+	// Scrub tokens from fixedBody
+	for user, token := range s.Config.TokenMap() {
+		t := []byte(token)
+		if bytes.Contains(fixedBody, t) {
+			fixedBody = bytes.Replace(fixedBody, t, []byte(user), -1)
```

Comment:
@sheax0r I consider replacing the token with something like `[]bytes(user + ":token")` for a little more annotation. Thoughts?

Reply:
Or even `[]bytes("\"replaced " + user + " token\")`... but didn't want to go crazy. =P